KsApi/models/UpdateUserEnvelope.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
 
-public struct UpdateAccountEnvelope: Decodable {
+public struct UpdateUserEnvelope: Decodable {
```

Comment:
Oh was this just a rename??

Reply:
Yea it was just a rename, I should've pointed that out...made more sense because we can use `UpdateUserEnvelope` for both return of `UpdateUserAccountMutation` and `UpdateUserProfileMutation`.

Reply:
Sounds good.